govcd/vdc_test.go

```diff
@@ -561,3 +561,54 @@ func assertVDCAccessSettings(wanted, received []*types.AccessSetting) error {
 	}
 	return nil
 }
+
+// TestVAppTemplateRetrieval tests that VDC receiver objects can search vApp Templates successfully.
+func (vcd *TestVCD) TestVAppTemplateRetrieval(check *C) {
```

Comment:
could you possibly convert it to use nsxt vdcs and templates, just so we have one less thing to fix when removing nsxv and testing CDS

Reply:
Done, I've picked your proposed changes from [here](https://github.com/vmware/go-vcloud-director/pull/509/files#diff-ccdc0aa4095bdd61b24c396c0572e6bcb8933ca63a89e9e1e5b9d7cdcff4e5bfR144), I also modified `sample_govcd_test_config.yaml` (let me know your thoughts about it, since probably you also need it in your PR, to avoid conflicts).

Reply:
Thanks!